embedchain/eval/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .base import BaseMetric  # noqa: F401
+from .config import EvalConfig  # noqa: F401
+from .metrics.context_relevancy import ContextRelevance  # noqa: F401
+from .utils import EvalData, EvalMetric  # noqa: F401
```

Comment:
We don't want to expose all of these modules at the top level so can remove most of these.